README-CONAN.md

```diff
@@ -289,6 +289,9 @@ If you wish to use the static C run-time library, use the following option in th
 |:---                  |:---------           |:-------------------|
 | **profile setting**  | compiler.runtime=MT | compiler.runtime=MTd |
 
+If you receive a linker warning concerning LIBCMT, it is because you are attempting to link libraries which have been built with different run-time libraries.
+You should do everything with dynamic or static run-time.  It is OK to build static libraries that use the dynamic run-time library.
```

Comment:
```suggestion
You should build everything using the dynamic or static run-time. Note, that it is possible to build static libraries that use the dynamic run-time library.
```

Reply:
Can we say "You should link everything with the dynamic or static run-time. You can link a static library with the dynamic run-time if you wish."

Reply:
Updated my comment accordingly.